prime-angular-frontend/src/app/shared/components/dialogs/content/send-email/send-email.component.html

```diff
@@ -0,0 +1,38 @@
+<h2 mat-dialog-title
+    class="d-flex align-items-stretch">
+  <mat-icon [color]="'primary'">help</mat-icon>
+  <span class="flex-grow-1">
+    {{ this.title }}
+  </span>
+</h2>
+
+<mat-dialog-content>
+  <span>Send email to:</span>
+
+  <div class="row mt-4">
+    <div class="col-auto">
+      <button mat-stroked-button
+              cdkFocusInitial
+              (click)="onCancel()">Cancel</button>
+    </div>
+    <div class="col-auto">
+      <mat-card>
```

Comment:
The equivalent drop down menu for `Send To` is implemented in the `AdjudicationActionsComponent` where `Adjudicator Actions` is a `mat-flat-button` with a `mat-menu` trigger.

Reply:
Changed the "Send To" to a drop down menu button